services/controller/standard/events.go

```diff
@@ -343,35 +345,53 @@ func (s *Service) VerifySyncCommitteeMessages(ctx context.Context, data *apiv1.H
 	_, span := otel.Tracer("attestantio.vouch.services.controller.standard").Start(ctx, "VerifySyncCommitteeMessages")
 	defer span.End()
 
-	logger := log.With().Uint64("slot", uint64(data.Slot)).Logger()
+	// We verify against the previous slot as that is when the sync committee will have reported.
+	previousSlot := data.Slot - 1
+	currentSlot := data.Slot
+
+	// Logging with the current slot as that is when this code is executed.
+	logger := log.With().Uint64("slot", uint64(currentSlot)).Logger()
 	logger.Trace().Msg("Received head event")
 
-	rootReported, found := s.syncCommitteeMessenger.GetBeaconBlockRootReported(data.Slot)
+	previousSlotData, found := s.syncCommitteeMessenger.GetDataUsedForSlot(previousSlot)
 	if !found {
-		logger.Trace().Msg(fmt.Sprintf("no reported sync committee message data for slot: %d, skipping validation", data.Slot))
+		logger.Trace().Msg(fmt.Sprintf("No reported sync committee message data for slot: %d, skipping validation", previousSlot))
```

Comment:
As we are using structured logging it makes sense to keep variables out of the message.  So, for example, this would look like:

```
logger.Trace().Uint64("slot",uint64(previousSlot)).Msg("No reported sync committee message data for slot; skipping validation")
```

Please update similarly throughout.